src/context/GeoStylerContext/GeoStylerContext.tsx

```diff
@@ -56,10 +47,17 @@ export type CompositionContext = {
   Style?: StyleComposableProps;
 };
 
+export interface UnsupportedPropertiesContext extends UnsupportedProperties {
+  options?: {
+    hideUnsupported?: boolean;
+    locale?: GeoStylerLocale['UnsupportedPropertiesUtil'];
```

Comment:
Why is this not part of the other locale?

Reply:
The "other" locale is not yet supported.